docs/data/data-grid/tree-data/TreeDataLazyLoading.tsx

```diff
@@ -1,307 +1,51 @@
-// TODO rows v6: Adapt to new lazy loading api
 import * as React from 'react';
```

Comment:
The next step for this demo is to support server-side sorting and filtering.
When using server-side data fetching, the sorting and filtering should be automatically in “server” mode, and the models should be passed to the function that fetches the rows.